### ISSUE TITLE
Interactive Legend is Unresponsive

### ISSUE THREAD

**Issue by hwilliams10** [altair-viz/altair#2163]
Hi,

I've created plots with interactive legends that were working yesterday, but the interactivity appears to be completely unresponsive today. 

Running the code in the examples gallery also fails to generate an interactive legend for me locally:
```import altair as alt
from vega_datasets import data

source = data.unemployment_across_industries.url

selection = alt.selection_multi(fields=['series'], bind='legend')

alt.Chart(source).mark_area().encode(
    alt.X('yearmonth(date):T', axis=alt.Axis(domain=False, format='%Y', tickSize=0)),
    alt.Y('sum(count):Q', stack='center', axis=None),
    alt.Color('series:N', scale=alt.Scale(scheme='category20b')),
    opacity=alt.condition(selection, alt.value(1), alt.value(0.2))
).add_selection(
    selection
)
```

For reference I am using Altair version 4.1.0 and working in Jupyter notebook.

Any help would be greatly appreciated!

**Comment by annamay33** [comment]
Hello,

I have the same problem, the interactive plots I created and saved as .html last week stopped being responsive when I open them in my browser.

I am using the same Altair version: 4.1.0.

Thank you!

**Comment by jakevdp** [comment]
It looks like this was caused by the Vega 5.12.1 release. It works if I pin the vega JS version to 5.12.0

**Comment by jakevdp** [comment]
Reported in https://github.com/vega/vega/issues/2641

**Comment by annamay33** [comment]
Thank you @jakevdp! Saving the charts with `vega_version='5.12.0'` made them work again.

**Comment by jakevdp** [comment]
Vega has released [5.12.2](https://www.jsdelivr.com/package/npm/vega) with a fix, so the workaround should no longer be necessary.

**Comment by annamay33** [comment]
I can still reproduce the error when the parameter `vega_version='5.12.0' ` isn't set.

**Comment by jakevdp** [comment]
It’s probably due to your browser cacheing the old library version.

**Comment by firasm** [comment]
@hwilliams10 This issue should be closed as the 5.12.2 fix resolves the error.

**Comment by eitanlees** [comment]
I wanted to make a note about caching issues. I found problems persisted on certain pages even after attempting to clear cached files in chrome. I learned that if you open the developer console and right click on the refresh button, you can select `Empty Cache and Hard Reload`. That has fixed any problems I have encountered so far. ([link for more info](https://www.getfilecloud.com/blog/2015/03/tech-tip-how-to-do-hard-refresh-in-browsers/))
